metadata-ingestion/src/datahub/ingestion/transformer/base_transformer.py

```diff
@@ -235,7 +258,12 @@ def transform(
                                     aspectName=self.aspect_name(),
                                     aspect=transformed_aspect,
                                 ),
-                                metadata=record_metadata,
+                                metadata=envelope.metadata,
                             )
+                        yield from self._handle_end_of_stream(
```

Comment:
this should be after the `self._mark_processed(urn)` call and two level of indentation less

Reply:
done